src/libraries/System.Security.Cryptography.Xml/tests/DSAKeyValueTest.cs

```diff
@@ -51,39 +51,42 @@ public void Ctor_Dsa_Null()
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/20575", TestPlatforms.OSX)]
         public void GetXml()
         {
-            DSAKeyValue dsa = new DSAKeyValue();
-            XmlElement xmlkey = dsa.GetXml();
+            using (DSA dsaKey = DSA.Create())
+            {
+                dsaKey.ImportPkcs8PrivateKey(TestHelpers.DsaPkcs8Key, out _);
+                DSAKeyValue dsa = new DSAKeyValue(dsaKey);
+                XmlElement xmlkey = dsa.GetXml();
 
-            XmlNamespaceManager ns = new XmlNamespaceManager(xmlkey.OwnerDocument.NameTable);
-            ns.AddNamespace("schema", SignedXml.XmlDsigNamespaceUrl);
+                XmlNamespaceManager ns = new XmlNamespaceManager(xmlkey.OwnerDocument.NameTable);
+                ns.AddNamespace("schema", SignedXml.XmlDsigNamespaceUrl);
 
-            IEnumerable<XmlNode> elements =
-                new[] { "P", "Q", "G", "Y", "J", "Seed", "PgenCounter" }
-                .Select(elementName => xmlkey.SelectSingleNode($"/schema:DSAKeyValue/schema:{elementName}", ns))
-                .Where(element => element != null);
+                IEnumerable<XmlNode> elements =
+                    new[] { "P", "Q", "G", "Y", "J", "Seed", "PgenCounter" }
+                    .Select(elementName => xmlkey.SelectSingleNode($"/schema:DSAKeyValue/schema:{elementName}", ns))
+                    .Where(element => element != null);
 
-            //There MUST be existing elements
-            Assert.NotEmpty(elements);
+                //There MUST be existing elements
+                Assert.NotEmpty(elements);
 
-            //Existing elements MUST include a "Y"-Element
-            Assert.True(elements.SingleOrDefault(element => element.Name == "Y") != null);
+                //Existing elements MUST include a "Y"-Element
+                Assert.True(elements.SingleOrDefault(element => element.Name == "Y") != null);
 
-            //Existing elements MUST contain InnerText
-            Assert.True(elements.All(element => !string.IsNullOrEmpty(element.InnerText)));
+                //Existing elements MUST contain InnerText
+                Assert.True(elements.All(element => !string.IsNullOrEmpty(element.InnerText)));
 
-            //Existing elements MUST be convertible from BASE64
-            elements.Select(element => Convert.FromBase64String(element.InnerText));
+                //Existing elements MUST be convertible from BASE64
+                elements.Select(element => Convert.FromBase64String(element.InnerText));
```

Comment:
Heh. This doesn't do anything I think because Linq lazy evaluates. I'll fix this up.